plugins/fastboot/README.md

```diff
@@ -0,0 +1,24 @@
+Fastboot Support
+================
+
+Introduction
+------------
+
+This plugin is used to update hardware that uses the fastboot protocol.
+
```

Comment:
I think the documentation will need to include more about how to build CAB files for this.  They're a bit special with the introduction of the extra files in the archive and XML nodes to refer to them.

Reply:
I think this is true of all the plugins, and would be a helpful section regardless; some require Intel HEX, some require SREC, some binary etc. What about this:

    Firmware Format
    ---------------

    The daemon will decompress the cabinet archive and extract a firmare blob in
    ZIP file format. Inside the zip file must be all the firmware images for each
    partition and a manifest file. The partition images can be in any format, but
    the manifest must be either an Android `flashfile.xml` format file, or a QFIL
    `partition_nand.xml` format file.

    For both types, all partitions with a defined image found in the zip file will
    be updated.

Reply:
That's perfect